ACHNBrowserUI/ACHNBrowserUI.xcodeproj/project.pbxproj

```diff
@@ -1234,6 +1267,8 @@
 				687B5768244A030700C1E86F /* TurnipsView.swift in Sources */,
 				6951837224598C8A005D28E0 /* NotificationsManager.swift in Sources */,
 				4C7FE78324574FB50011E8AB /* ActivityIndicator.swift in Sources */,
+				6942FC37244988F80062FE8F /* SafariView.swift in Sources */,
```

Comment:
@Dimillian do you have an idea why it's here?
I had to remove a conflict, so maybe I did something wrong by reintroduce it?

Reply:
I moved it 2 times. So it have to be within the App it can't be in UI package unfortunately as SafariService and UIKit can't be compiled in a package standalone...

Reply:
OK I will let it like this, if we see that a red file about that appears in Xcode, we can safely remove it. (I didn't see such thing by now)